### DISCUSSION TITLE
چطور پنل هیدیفای را پاک کنم

### DISCUSSION THREAD

**Discussion by sepehrgh8090** [hiddify/Hiddify-Manager#3268] [Q&A سوالات رایج]
سلام دوستان و عزیزان 
در تمام متنهای آموزشی اشاره مشخص به راه اندازیی پنل و دستورات مربوطه شده اما من هیج جا ندیدم برای اینکه این پنل را از روی سرور پاک کنیم چه دستوری باید بزنیم
ممنون میشم راهنمایی کنید

**Comment by Plant1000** [comment]
سلام
باید از طریق طریق منوی ssh این کار رو انجام بدید

**Comment by sepehrgh8090** [comment]
از منو میزنم پاک نمیشه
گفتم شاید با دستور مستقیم بشه
